Makefile

```diff
@@ -12,7 +15,11 @@ docs:
 test:
 	go test ./... -v
 
+thirdparty-licenses:
+	go install github.com/google/go-licenses
```

Comment:
this didn't work for me:

```
go install github.com/google/go-licenses
no required module provides package github.com/google/go-licenses; to add it:
	go get github.com/google/go-licenses
make: *** [thirdparty-licenses] Error 1
```

I would suggest either removing the `go install` from the Makefile target, either adding all the commands for it to work

Reply:
Added `go get`